azavea/raster-foundry
# Feature/lol/s3 uuid

* Files uploaded to S3 will now be named {user_id}-{uuid}
 * The api checks to ensure the front end user matches the user making the request.
 * Django image model updated to allow for filename and uuid data.

Connects #46 

To test:
  * Log in.
  * Attempt to upload a file by going to the file uploader and adding one or more images.
  * Check s3 to see if they've appeared.
  * This should not be possible while logged out.
  * Attempting to fake the request by manually visiting the api endpoint and using the wrong user id should fail.
